src/dsp_tools/cli/create_parsers.py

```diff
@@ -236,6 +237,23 @@ def _add_upload_files(
     subparser.add_argument("xml_file", help="path to XML file containing the data")
 
 
+def _add_ingest_files(
+    subparsers: _SubParsersAction[ArgumentParser],
+    default_dsp_api_url: str,
+    root_user_email: str,
+    root_user_pw: str,
+) -> None:
+    subparser = subparsers.add_parser(
+        name="ingest-files",
+        help="Experimental: ingest uploaded files on a DSP server and retrieve the mapping CSV file",
+    )
+    subparser.set_defaults(action="ingest-files")
+    subparser.add_argument("-s", "--server", default=default_dsp_api_url, help=dsp_server_text)
+    subparser.add_argument("-u", "--user", default=root_user_email, help=username_text)
+    subparser.add_argument("-p", "--password", default=root_user_pw, help=password_text)
```

Comment:
*thought [security]*: Passing a password as argument is not safe

Providing credentials as a command line argument is convenient but is considered to be security risk. This is because the arguments end up in the history of a shell, which is generally a bad design.

Other ways of providing the credentials might be:
* Request it to from the user in interactive mode
* Use a dot file for configuration
* Environment variables